vowpalwabbit/slim/src/model_parser.cc

```diff
@@ -1,5 +1,7 @@
 #include "model_parser.h"
 
+#include <vw_exception.h>
```

Comment:
It brings in `_UNUSED` to silence an unused variable warning

Reply:
ok, just seems a bit counter-intuitive given that slim explicitly doesn't use exceptions (even though this is safe since slim compiles out anything that throws), but seems ok

Reply:
just `((void)(var))`?